harvardcards/static/js/modules/collections-index.js

```diff
@@ -1,11 +1,30 @@
-define(['jquery', 'components/slider/CollectionSlider', 'utils/utils'], function($, CollectionSlider, utils) {
+define(['jquery', 'components/slider/CollectionSlider', 'utils/utils', 'jqueryui'], function($, CollectionSlider, utils) {
 	return {
 		initModule: function(){
-			$('.slider').each(function(index, el) {
-				var slider = new CollectionSlider(el);
-			});
 
-			utils.setupConfirm();
+
+            $('.courseHeader').click(function() {
+                var el = this;
+                $(this).next().toggle('slow', function(){
+                    var collection_id = $(el).data('collection-id');
+                    if($(el).next().is(":visible")){
+                        localStorage.setItem('show-collection-id'+collection_id, true);
```

Comment:
I was having issues with using key-value pairs in localStorage.. but I was doing it directly as opposed to creating the dict and then just including that -- the main issue is I'd probably have to remember to `JSON.parse()` after I get the result, but I'll give it another shot.